x-pack/metricbeat/module/aws/sqs/_meta/docs.asciidoc

```diff
@@ -18,3 +18,19 @@ sqs:ListQueues
 The aws sqs metricset comes with a predefined dashboard. For example:
 
 image::./images/metricbeat-aws-sqs-overview.png[]
+
+[float]
+=== Configuration example
+[source,yaml]
+----
+- module: aws
+  period: 300s
+  metricsets:
+    - sqs
+  access_key_id: '${AWS_ACCESS_KEY_ID:""}'
+  secret_access_key: '${AWS_SECRET_ACCESS_KEY:""}'
+  session_token: '${AWS_SESSION_TOKEN:""}'
+  default_region: '${AWS_REGION:us-west-1}'
```

Comment:
We have these settings all along the place, are these environment variables also used by the underlying libraries even if they are not set in this config? 

Then maybe for readability of the examples we could consider omitting them, and keep them only in the reference config. We can also maybe have a single section or page where we explain how to configure aws settings.

Reply:
Yes, I will move credential related documentations to a separate config with https://github.com/elastic/beats/pull/12727